README.md

```diff
@@ -27,7 +27,6 @@ After you're finished please remove all the comments and instructions!
 
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
   <h3><b>Microverse README Template</b></h3>
```

Comment:
[OPTIONAL] Unfortunately the previous reviewer should have invalidate your Pull Request since you have an touched README Template. You need to customize your README file with your custom content. You used [Microverse Readme template](https://github.com/microverseinc/readme-template) but make sure all the main sections present in yours and placeholders are replaced with your project and personal data. Please remove all the instructions targeting you. For example: "Describe how a new developer could make use of your project." The only optional section is the FAQ. Please remove the template logo on the top, if you have your own logo you can replace it with this one, otherwise please delete it. This is crucial to have a professional and instructive repository. 👌 
![Screen Recording 2023-03-24 at 10 18 10 PM](https://user-images.githubusercontent.com/25887253/227621296-953f5feb-16ca-4210-bc45-6e417d6db8bd.gif)